app/controllers/version_controller.rb

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2015 The Regents of the University of Michigan.
```

Comment:
At SUL-DLSS, the LICENSE at the root of the project is assumed to cover all original code in the project.  AFAIK, there are no additional requirements for headers in each file (at Stanford).